src/components/ChatWindow/ChatWindow.tsx

```diff
@@ -21,10 +21,11 @@ import useUuid from "@/hooks/useUuid"
 import SendBtn from "../SendBtn"
 
 const scriptTag = document.currentScript as HTMLScriptElement
-const botId = scriptTag?.getAttribute("botId") ?? ""
-
-console.log("Script tag is:", scriptTag)
-console.log("botId is: ", botId)
+const botId = scriptTag?.getAttribute("botId") ?? import.meta.env.VITE_BOT_ID // If botId is not provided as attribute in script tag, use the botId from environment variables
```

Comment:
```suggestion
const botId = scriptTag?.getAttribute("botId") ?? import.meta.env.VITE_DEV_BOT_ID // If botId is not provided as attribute in script tag, use the botId from environment variables
```

Can you help change to VITE_DEV_BOT_ID, please? so it's meant for development purpose only